x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/trigger/schedule/tool/CronEvalToolTests.java

```diff
@@ -20,4 +24,28 @@ public void testParse() throws Exception {
         String output = execute(countOption, Integer.toString(count), "0 0 0 1-6 * ?");
         assertTrue(output, output.contains("Here are the next " + count + " times this cron expression will trigger"));
     }
+
+    public void testGetNextValidTimes() throws Exception {
+        {
```

Comment:
for readability, just put `final int year = ZonedDateTime.now(ZoneOffset.UTC).getYear() + 1;` at the top of the test and use that one maybe?